### PR TITLE
docs: update input/url

### PR DESCRIPTION
**Pull request by jasonren0403**
### Description

Remove {{page}} macro and translated remaining EN content on page

### CODE REVIEW

File: files/zh-cn/web/html/element/input/url/index.md [hunk 1]
```diff
@@ -5,332 +5,379 @@ slug: Web/HTML/Element/Input/url
...
-   <td><strong><a href="#value">Value</a></strong></td>
-   <td>A {{domxref("DOMString")}} representing a URL, or empty</td>
+   <td><strong><a href="#值">值</a></strong></td>
+   <td>代表 URL 的字符串，或为空</td>
   </tr>
   <tr>
-   <td><strong>Events</strong></td>
-   <td>[`change`](/zh-CN/docs/Web/API/HTMLElement/change_event) and [`input`](/zh-CN/docs/Web/API/HTMLElement/input_event)</td>
+   <td><strong>事件</strong></td>
+   <td>
```
Comment (yin1999) [inline files/zh-cn/web/html/element/input/url/index.md:24]:
```suggestion
   <td>
```

File: files/zh-cn/web/html/element/input/url/index.md [hunk 2]
```diff
@@ -5,332 +5,379 @@ slug: Web/HTML/Element/Input/url
...
-| [`spellcheck`](#spellcheck)   | Controls whether or not to enable spell checking for the input field, or if the default spell checking configuration should be used |
+### list
 
-{{page("/en-US/docs/Web/HTML/Element/input/text", "list", 0, 1, 2)}}
+{{htmlattrdef("list")}} 属性指定了一个 {{HTMLElement("datalist")}} 元素的 id，用来提供输入建议。{{HTMLElement("datalist")}} 提供了一个预定义的值列表，向用户建议这个输入。列表中任何与 {{htmlattrxref("type", "input")}} 不兼容的值都不包括在建议选项中。所提供的值是建议，不是要求：用户可以从这个预定义的列表中选择，或者提供不同的值。
 
 ### maxlength
 
-The maximum number of characters (as UTF-16 code units) the user can enter into the `url` input. This must be an integer value 0 or higher. If no `maxlength` is specified, or an invalid value is specified, the `url` input has no maximum length. This value must also be greater than or equal to the value of `minlength`.
+用户可以输入到 `url` 输入框中的最大字符数（以 UTF-16 码点为单位）。必须为大于等于 0 的整数。如果未指定 `maxlength` 或指定了无效的值，则 `url` 输入框将没有最大值。这个值也必须大于等于 `minlength` 的值。
```
Comment (yin1999) [inline files/zh-cn/web/html/element/input/url/index.md:66]: One question here, should it be `代码单元`?

```suggestion
用户可以输入到 `url` 输入框中的最大字符数（UTF-16 代码单元）。必须为大于等于 0 的整数。如果未指定 `maxlength` 或指定了无效的值，则 `url` 输入框将没有最大值。这个值也必须大于等于 `minlength` 的值。
```

Comment (jasonren0403) [reply files/zh-cn/web/html/element/input/url/index.md:66]: 🤔 yeah, here it should be `代码单元`

File: files/zh-cn/web/html/element/input/url/index.md [hunk 3]
```diff
@@ -5,332 +5,379 @@ slug: Web/HTML/Element/Input/url
...
 
-The input will fail [constraint validation](/zh-CN/docs/Web/Guide/HTML/HTML5/Constraint_validation) if the length of the text value of the field is greater than `maxlength` UTF-16 code units long. Constraint validation is only applied when the value is changed by the user.
+如果文本框中的字符数大于 `maxlength` UTF-16 编码单元长度，则输入将无法通过[约束验证](/zh-CN/docs/Web/Guide/HTML/Constraint_validation)。约束验证仅作用于用户修改输入值的时候。
 
 ### minlength
 
-The minimum number of characters (as UTF-16 code units) the user can enter into the `url` input. This must be an non-negative integer value smaller than or equal to the value specified by `maxlength`. If no `minlength` is specified, or an invalid value is specified, the `url` input has no minimum length.
+用户可以输入到 `url` 输入框中的最小字符数（以 UTF-16 码点为单位）。该值必须是小于等于 `maxlength` 指定的值的非负整数值。如果未指定 `minlength` 或指定了无效的值，则 `url` 输入将没有最小值。
```
Comment (jasonren0403) [inline files/zh-cn/web/html/element/input/url/index.md:72]:
```suggestion
用户可以输入到 `url` 输入框中的最小字符数（以 UTF-16 代码单元为单位）。该值必须是小于等于 `maxlength` 指定的值的非负整数值。如果未指定 `minlength` 或指定了无效的值，则 `url` 输入将没有最小值。
```

File: files/zh-cn/web/html/element/input/url/index.md (hunk 2 above)
Comment (jasonren0403) [reply files/zh-cn/web/html/element/input/url/index.md:66]:
```suggestion
用户可以输入到 `url` 输入框中的最大字符数（以 UTF-16 代码单元为单位）。必须为大于等于 0 的整数。如果未指定 `maxlength` 或指定了无效的值，则 `url` 输入框将没有最大值。这个值也必须大于等于 `minlength` 的值。
```

**Review by yin1999 — APPROVED** [review]
Thanks

**Pull request merged**